tests/test_fastapi.py

```diff
@@ -124,11 +132,9 @@ def hello():
     assert fastapi_colon_request_exit_log["http.response.status_code"] == 500
 
 
-def test_alternate_installation(caplog):
+def test_alternate_installation(client, caplog):
```

Comment:
In streamlining the other tests, I believe you may have changed what this test was testing-- I wanted to confirm that `app.add_middleware()` resulted in the same behavior as `chipperize()`-- aside from the incorrect timing issue. With these changes, this test is no longer doing that.

Reply:
You're right - thanks - correcting.